src/features/session_replay/aggregate/index.js

```diff
@@ -22,38 +22,41 @@ import { AggregateBase } from '../../utils/aggregate-base'
 // would be better to get this dynamically in some way
 export const RRWEB_VERSION = '2.0.0-alpha.8'
 
+export const avgCompression = 0.12
+
 let recorder, gzipper, u8
 
 /** The "mode" with which the session replay is recording */
-const MODE = {
+export const MODE = {
   OFF: 0,
   FULL: 1,
   ERROR: 2
 }
```

Comment:
Can we move constants to a `constants.js` in the root of the feature?

Reply:
I think Chun actually moved this in his PR to the session manager.  Will reflect once the dust settles